HandmadeMath.h

```diff
@@ -617,6 +638,36 @@ HMM_TanF(float Radians)
 }
 
 HINLINE float
+HMM_ATanF(float Theta)
+{
+    float U = Theta*Theta;
+    float U2 = U*U;
+    float U3 = U2*U;
+    float U4 = U3*U;
+    float F = 1.0f + 0.33288950512027f*U - 0.08467922817644f*U2 + 0.03252232640125f*U3 - 0.00749305860992f*U4;
+    return Theta / F;
+}
+
+HINLINE float
+HMM_ATanF2(float Theta, float Theta2)
+{
+    if (HMM_ABS(Theta2) > HMM_ABS(Theta)) {
+        float A = HMM_ATanF(Theta / Theta2);
+        if (Theta2 > 0.0f)
+            return A;
+        else
+            return Theta > 0.0f ? A + HMM_PI : A - HMM_PI;
+    }
+    else {
+        float A = HMM_ATanF(Theta2 / Theta);
+        if (Theta2 > 0.0f)
+            return Theta > 0.0f ? (HMM_PI/2) - A : - (HMM_PI/2) - A;
+        else 
+            return Theta > 0.0f ? (HMM_PI/2) + A : - (HMM_PI/2) + A;
+    }
+}
```

Comment:
Im really not sure about this, do you have any opinions i mean if these weren't that hard to implement is it a bad idea to pull them in ?